multigeiger/display.cpp

```diff
@@ -106,28 +106,28 @@ void DisplayGMC(int TimeSec, int RadNSvph, int CPS, bool use_display, bool conne
   if (!isLoraBoard) {
     char output[80];
     int TimeMin = TimeSec / 60;         // calculate number of minutes
-    if (TimeMin >= 999) TimeMin = 999;  // limit minutes to max. 999
+    TimeMin = TimeMin % 1000;  // limit minutes to max. 999, roll over
 
     // print the upper line including time and measured radation
+    pu8x8->setFont(u8x8_font_open_iconic_embedded_1x1);
+    pu8x8->print(ble_active ? '\x4A' : '\x20');
```

Comment:
4A is 'J'? If so, we could also write 'J' there.

Reply:
Not in the selected font, there it represents a Bluetooth symbol. I've added a comment and changed '\x20' to ' '